timeutil/proto.go

```diff
@@ -11,47 +11,47 @@ import (
 )
```

Comment:
we can remove this file completely. If we are going to modify the interface than any consumer could just upgrade to the latest proto and use the functions that come with it rather than these.